slackify-html.js

```diff
@@ -175,6 +174,10 @@ function walk(dom, nesting) {
                     content.charAt(content.length - 1) === '*') {
                   innerOutput += content;
                 }
+                else if (el.name === 'h1' || el.name === 'h2' || el.name === 'h3' || el.name === 'h4') {
```

Comment:
would moving this to the line after 151 do the same thing?

Reply:
@austinparkk do you mean here? 
```
case 'h1':
case 'h2':
case 'h3':
case 'h4':
```

Reply:
yea

Reply:
gotcha, yeah that does seem to work - i'll push up the changes!

Reply:
oh you can just put the code under the h4 case i think and it will work for h1,h2,h3,h4 instead of repeating the code.

Reply:
moved it back since this does affect the whitespacing as James had mentioned

Reply:
i think this should be the first check in this if block.

Reply:
(i believe this code will fail for <h1><b>some title</b></h1>)

Reply:
@jahlbornG i have been able to see if fail for `<h1>*asterisk title with *bold**</h1>` but succeed when swapped so i'll be switching the order. nice catch!